app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+  before_action :authorized?
+  private
+  def authorized?
+    unless user.admin?
+      flash[:error] = "You are not authorized to view that page."
```

Comment:
localize

Reply:
localized, but i don't know how to do lazy in controller with flash[:error].